common/app/views/fragments/contentMeta.scala.html

```diff
@@ -41,13 +41,20 @@
             }
             @fragments.meta.contactAuthor(item.tags, amp)
         }
+
+        @if((item.trail.shouldHidePublicationDate || item.content.isGallery)) {
```

Comment:
This fragment should rather be:
```scala
@if(ActiveExperiments.isParticipating(experiments.Garnett)) {
  @if(!(item.trail.shouldHidePublicationDate || item.content.isGallery)) {
    @fragments.meta.dateline(item.trail.webPublicationDate, item.fields.lastModified, item.content.hasBeenModified, item.fields.firstPublicationDate, item.tags.isLiveBlog, item.fields.isLive)
  }
}
```

Reply:
aha, thanks. I need to import "@import experiments._" at the top too right?

Reply:
oh yeah sorry, totally forgot that.